dali/util/local_file.cc

```diff
@@ -36,18 +38,20 @@
 static int _sysctl(struct __sysctl_args *args);
 
 static int get_max_vm_cnt() {
-  struct __sysctl_args args = {0, };
   int vm_cnt;
   size_t vm_cnt_sz;
+#if !defined(__AARCH64_QNX__) && !defined(__AARCH64_GNU__)
   int name[] = { CTL_VM, VM_MAX_MAP_COUNT };
+  struct __sysctl_args args = {0, };
 
   args.name = name;
   args.nlen = sizeof(name)/sizeof(name[0]);
   args.oldval = &vm_cnt;
   args.oldlenp = &vm_cnt_sz;
+#endif
```

Comment:
Why to have this #endif here? I think this can be one block.

Reply:
I wanted to disturb the x86_64 code flow as least as possible. Will reorganize this function

Reply:
fixed